Synthesis failed for Google.Identity.AccessContextManager.V1

Hello! Autosynth couldn't regenerate Google.Identity.AccessContextManager.V1. :broken_heart:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Identity.AccessContextManager.V1 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
8b3eedfe
2021-07-27 23:04:41,930 autosynth [DEBUG] > Running: git log -1 --pretty=%at b1844c9b9cd5fbaf53068555e2b9ffe942f3c1c4
2021-07-27 23:04:41,934 autosynth [DEBUG] > Running: git log -1 --pretty=%at 11d5ead7457a84d64653818d03bd45ad7bb8def7
2021-07-27 23:04:41,938 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3744b86f39039fccaf7b2276db191d21f118f2b4
2021-07-27 23:04:41,942 autosynth [DEBUG] > Running: git log -1 --pretty=%at b65a2dd5573b76dc40bbc4487966232bfec09f7d
2021-07-27 23:04:41,945 autosynth [DEBUG] > Running: git log -1 --pretty=%at e9ee226dfde10a831fa82c409c76b1359523c27d
2021-07-27 23:04:41,949 autosynth [DEBUG] > Running: git log -1 --pretty=%at a5eda9f0e1a590ed753c134bf3da09836618646f
2021-07-27 23:04:41,953 autosynth [DEBUG] > Running: git log -1 --pretty=%at 762094a99ac6e03a17516b13dfbef37927267a70
2021-07-27 23:04:41,957 autosynth [DEBUG] > Running: git log -1 --pretty=%at 746461e4242e8d8bf2e2e78994784dba55c5ec31
2021-07-27 23:04:41,961 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6cf3b8e73ffae3114b041512fedb29fa1b883b75
2021-07-27 23:04:41,965 autosynth [DEBUG] > Running: git log -1 --pretty=%at c8fac9a3f5eddc3240b596c2d93183a457d77d16
2021-07-27 23:04:41,968 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3c7864b59c9b80503076db7c2aaa6cd68921e3ce
2021-07-27 23:04:41,972 autosynth [DEBUG] > Running: git log -1 --pretty=%at 28ccba2abfee4d164a78a20546478cbe266188a3
2021-07-27 23:04:41,975 autosynth [DEBUG] > Running: git log -1 --pretty=%at fe82a7539e9df02e96e632f5362c1092c95ba29b
2021-07-27 23:04:41,979 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9cd96aea622ce4c21d3c0bcd7cb788bc39f65a31
2021-07-27 23:04:41,982 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5bf36f20f8e49c18620bf20a45645310f5bf6950
2021-07-27 23:04:41,986 autosynth [DEBUG] > Running: git log -1 --pretty=%at c3b8283ae6a67b575ee863764312645ec8b2d60f
2021-07-27 23:04:41,990 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4fca877897a4639796bba0731c6f579724d80d72
2021-07-27 23:04:41,993 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8962c92e97495d0795b427d4aa4326b0d06e33eb
2021-07-27 23:04:41,997 autosynth [DEBUG] > Running: git log -1 --pretty=%at b69daf76f9991defa8435acb9114df3e6764f2b1
2021-07-27 23:04:42,001 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2948f31dd210514f87ce6e1e08bce48ecf4660d1
2021-07-27 23:04:42,004 autosynth [DEBUG] > Running: git log -1 --pretty=%at 46fdf2d70c3c498b82ef8a05172933674f977653
2021-07-27 23:04:42,008 autosynth [DEBUG] > Running: git log -1 --pretty=%at b837e91381d0de7d6ac711cbc8f46d1c3f66bcf9
2021-07-27 23:04:42,012 autosynth [DEBUG] > Running: git log -1 --pretty=%at 0b261def1cb4c61a9ddbb8bf14b103ce6add1bce
2021-07-27 23:04:42,016 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6919bc1030d57680e76ef319013bc8e68c1fb7ce
2021-07-27 23:04:42,021 autosynth [DEBUG] > Running: git checkout 8d1fa3b953a019e31cd4b9f56a4fe51715bee6a2
Note: switching to '8d1fa3b953a019e31cd4b9f56a4fe51715bee6a2'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 8d1fa3b95 Release Google.Cloud.ServiceDirectory.V1Beta1 version 1.0.0-beta05
2021-07-27 23:04:42,048 autosynth [DEBUG] > Running: git checkout 6919bc1030d57680e76ef319013bc8e68c1fb7ce
Note: switching to '6919bc1030d57680e76ef319013bc8e68c1fb7ce'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 6919bc10 chore: regenerate API index
2021-07-27 23:04:42,064 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Identity.AccessContextManager.V1-570
2021-07-27 23:04:42,069 autosynth [DEBUG] > Running: git checkout autosynth-Google.Identity.AccessContextManager.V1-570
Switched to branch 'autosynth-Google.Identity.AccessContextManager.V1-570'
2021-07-27 23:04:42,096 autosynth [INFO] > Running synthtool
2021-07-27 23:04:42,096 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-07-27 23:04:42,096 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Identity.AccessContextManager.V1/570/sponge_log.log
2021-07-27 23:04:42,099 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-07-27 23:04:42,344 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Identity.AccessContextManager.V1/synth.py.
On branch autosynth-Google.Identity.AccessContextManager.V1-570
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Identity.AccessContextManager.V1 for compatibility checking
Generating Google.Identity.AccessContextManager.V1
Building new version of Google.Identity.AccessContextManager.V1 for compatibility checking
AccessContextManager.cs(280,588): error CS0234: The type or namespace name 'LongRunning' does not exist in the namespace 'Google' (are you missing an assembly reference?) [/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Identity.AccessContextManager.V1/Google.Identity.AccessContextManager.V1/Google.Identity.AccessContextManager.V1.csproj]
2021-07-27 23:04:48,345 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Identity.AccessContextManager.V1:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Identity.AccessContextManager.V1/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Identity.AccessContextManager.V1')' returned non-zero exit status 1.
2021-07-27 23:04:48,379 autosynth [ERROR] > Synthesis failed
2021-07-27 23:04:48,379 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 8d1fa3b95 Release Google.Cloud.ServiceDirectory.V1Beta1 version 1.0.0-beta05
2021-07-27 23:04:48,412 autosynth [DEBUG] > Running: git checkout autosynth-Google.Identity.AccessContextManager.V1
Switched to branch 'autosynth-Google.Identity.AccessContextManager.V1'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 356, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 336, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/8b7b770b-4ba5-4658-99c8-7a2d2bf3f2f5/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

--- a/README.md
+++ b/README.md
@@ -173,7 +173,7 @@ Each package name links to the documentation for that package.
 | [Google.Cloud.Workflows.V1](https://googleapis.dev/dotnet/Google.Cloud.Workflows.V1/1.0.0) | 1.0.0 | [Workflows (V1 API)](https://cloud.google.com/workflows/docs/apis) |
 | [Google.Cloud.Workflows.V1Beta](https://googleapis.dev/dotnet/Google.Cloud.Workflows.V1Beta/1.0.0-beta02) | 1.0.0-beta02 | [Workflows (V1Beta API)](https://cloud.google.com/workflows/docs/apis) |
 | [Google.Identity.AccessContextManager.Type](https://googleapis.dev/dotnet/Google.Identity.AccessContextManager.Type/1.1.0) | 1.1.0 | Version-agnostic types for the Google Identity Access Context Manager API |
-| [Google.Identity.AccessContextManager.V1](https://googleapis.dev/dotnet/Google.Identity.AccessContextManager.V1/1.2.0) | 1.2.0 | Protocol buffer types for the Google Identity Access Context Manager API V1 |
+| [Google.Identity.AccessContextManager.V1](https://googleapis.dev/dotnet/Google.Identity.AccessContextManager.V1/1.2.0) | 1.2.0 | Recommended Google client library to access the Identity Access Context Manager API V1 |
 | [Google.LongRunning](https://googleapis.dev/dotnet/Google.LongRunning/2.2.0) | 2.2.0 | Support for the Long-Running Operations API pattern |
 | [Grafeas.V1](https://googleapis.dev/dotnet/Grafeas.V1/2.2.0) | 2.2.0 | [Grafeas](https://grafeas.io/) |
 

--- a/apis/Google.Identity.AccessContextManager.V1/Google.Identity.AccessContextManager.V1/Google.Identity.AccessContextManager.V1.csproj
+++ b/apis/Google.Identity.AccessContextManager.V1/Google.Identity.AccessContextManager.V1/Google.Identity.AccessContextManager.V1.csproj
@@ -4,13 +4,14 @@
     <Version>1.2.0</Version>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Description>Protocol buffer types for the Google Identity Access Context Manager API V1.</Description>
+    <Description>Recommended Google client library to access the Identity Access Context Manager API V1.</Description>
     <PackageTags>identity;Google;Cloud</PackageTags>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="ConfigureAwaitChecker.Analyzer" Version="1.0.1" PrivateAssets="All" />
     <PackageReference Include="Google.Api.CommonProtos" Version="[2.3.0, 3.0.0)" />
     <PackageReference Include="Google.Identity.AccessContextManager.Type" Version="[1.1.0, 2.0.0)" />
+    <PackageReference Include="Google.LongRunning" Version="[2.2.0, 3.0.0)" />
     <PackageReference Include="Microsoft.DotNet.Analyzers.Compatibility" Version="0.2.12-alpha" PrivateAssets="All" />
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" />
     <Analyzer Condition="Exists('..\..\..\tools\Google.Cloud.Tools.Analyzers\bin\$(Configuration)\netstandard1.3\publish\Google.Cloud.Tools.Analyzers.dll')" Include="..\..\..\tools\Google.Cloud.Tools.Analyzers\bin\$(Configuration)\netstandard1.3\publish\Google.Cloud.Tools.Analyzers.dll" />

--- a/apis/apis.json
+++ b/apis/apis.json
@@ -3045,13 +3045,14 @@
       "protoPath": "google/identity/accesscontextmanager/v1",
       "version": "1.2.0",
       "targetFrameworks": "netstandard2.0;net461",
-      "description": "Protocol buffer types for the Google Identity Access Context Manager API V1.",
+      "description": "Recommended Google client library to access the Identity Access Context Manager API V1.",
       "tags": [
         "identity"
       ],
       "dependencies": {
         "Google.Api.CommonProtos": "2.3.0",
-        "Google.Identity.AccessContextManager.Type": "1.1.0"
+        "Google.Identity.AccessContextManager.Type": "1.1.0",
+        "Google.LongRunning": "2.2.0"
       }
     },
     {

